modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2268,6 +2269,83 @@ export class Wallet implements IWallet {
     }
   }
 
+  public async buildTokenEnablements(
+    params: BuildTokenEnablementOptions = { tokens: [] }
+  ): Promise<PrebuildTransactionResult[]> {
+    if (!this.baseCoin.requiresTokenEnablement()) {
+      throw new Error(`${this.baseCoin.getFullName()} does not require token enablements`);
+    }
+    if (params.tokens.length === 0) {
+      throw new Error('No tokens are being specified');
+    }
+    if (params.recipients) {
+      throw new Error('Can not specify recipients for token enablement transactions');
+    }
+
+    if (params.reqId) {
+      this.bitgo.setRequestTracer(params.reqId);
+    }
+
+    const buildParams = _.pick(params, this.prebuildWhitelistedParams());
```

Comment:
do we want a separate set of whitelisted params just for token enablement?

Reply:
`buildParams` here are the parameters that we plan on sending to `prebuildTransacation` so `prebuildWhitelistedParams` are the params that we want. We write token names into `recipients` later.

Reply:
Also, when we support the tss builds, we need to add the enable token params to `prebuildWhitelistedParams` for building with intent to work.